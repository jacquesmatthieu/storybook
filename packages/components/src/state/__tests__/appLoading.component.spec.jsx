import React from 'react';
import { shallow } from 'enzyme';

import AppLoading from '../appLoading.component';

describe('AppLoading', () => {
  let props;

  beforeEach(() => {
    props = {
      appId: 'foogere',
    };
  });

  const getAppLoadingWrapper = () => shallow(<AppLoading {...props} />);

  it('should render properly', () => {
    // When
    const wrapper = getAppLoadingWrapper();

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
