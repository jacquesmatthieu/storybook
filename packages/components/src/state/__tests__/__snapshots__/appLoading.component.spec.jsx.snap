// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLoading should render properly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "height": "100%",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <div
    style={
      Object {
        "maxWidth": "60rem",
      }
    }
  />
</div>
`;
